### PR TITLE
Fix pagination

### PR DESCRIPTION
**Pull request by micorix**
Fix for #45.

### CODE REVIEW

File: src/components/sections/SchoolsPage/Pagination.tsx [hunk 1]
```diff
@@ -39,37 +39,39 @@ const PaginationSkipButton = styled(BaseButton)`
   }
 `;
 
-const Pagination: FC<any> = ({ count, page, onPageChange }) => {
+const Pagination: FC<any> = ({ count, page, onPageChange, disabled }) => {
   const totalPages = getTotalPages(count);
 
+  if (totalPages === 1) return null;
```
Comment (mlazowik) [inline src/components/sections/SchoolsPage/Pagination.tsx:45]: So when there are no results it shows up, disabled, and if there's only one page it does now show up at all?

Comment (micorix) [reply src/components/sections/SchoolsPage/Pagination.tsx:45]: * When there is one page is does not show up at all
* When `data` variable is falsy the request is pending. (It's how SWR handles it. Like in this example -> https://swr.vercel.app/). Changed from `!schools` to `!data`. My bad.

**Review by mlazowik — APPROVED** [review]

File: src/pages/SchoolsGridPage.tsx [hunk 2]
```diff
@@ -59,12 +58,13 @@ const LoadingCard = styled(Card)`
...
   box-shadow: none;
+  margin-bottom: 3em;
   &::after {
     background: #eee;
   }
 `;
 
-const Results: FC<any> = ({ schools, error, page, count, onPageChange }) => {
+const Results: FC<any> = ({ schools, error }) => {
   if (error) return <ErrorInfo />;
```
Comment (MariaRukat) [inline src/pages/SchoolsGridPage.tsx:68]: This file is quite long, maybe we can move Results to a separate file? And all visual components could also be in the separate file

**Review by MariaRukat — CHANGES_REQUESTED** [review]
I've left a comment :)

**Review by MariaRukat — APPROVED** [review]

**Review by MariaRukat — APPROVED** [review]

**Pull request merged**